### ISSUE TITLE
Can't read the last 2 bytes from word peripheral space

### ISSUE THREAD

**Issue by tgtakaoka** [dlbeer/mspdebug#65]
`md` command can't read the last 2 bytes from word peripheral space, 0x100 <= addr < 0x200.

```
$ mspdebug sim
...
(mspdebug) simio add timer TA3
Added new device "TA3" of type "timer".
(mspdebug) mw 0x170 1 2 3 4 5 6 7 8
(mspdebug) md 0x170 0x10
    00170: 01 02 03 04 05 06 00 00 00 00 00 00 00 00 ff ff |................|
(mspdebug) md 0x170 0x6
    00170: 01 02 03 04 ff ff                               |......          |
(mspdebug) simio info TA3
Base address: 0x0160
IV address:   0x012e
IRQ0:         9
IRQ1:         8

TACTL:        0x0000
TAR:          0x0201
TAIV:         0x00

Channel  0, TACTL = 0x0000, TACCR = 0x0403
Channel  1, TACTL = 0x0000, TACCR = 0x0605
Channel  2, TACTL = 0x0000, TACCR = 0x0000
(mspdebug) 
```

**Comment by tgtakaoka** [comment]
This issue should be addressed by pull request #62.

**Comment by tgtakaoka** [comment]
Fixed.
